hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/steps/NodeStakeUpdates.java

```diff
@@ -173,14 +175,17 @@ public static boolean isNextStakingPeriod(
     }
 
     private void keyNewRoster(
-            @NonNull final HandleContext handleContext, @NonNull final WritableRosterStore rosterStore) {
+            @NonNull final HandleContext handleContext,
+            @NonNull final WritableRosterStore rosterStore,
+            @NonNull final TssMetrics tssMetrics) {
```

Comment:
Does it make sense to pass `TssMetrics` into `TssBaseService`, so we don't have to pass it through here? 🤔

Reply:
I've updated the branch in that way you can have a look @mhess-swl